Saved Queries: Pass editor ref for dynamic dropdown display (#114321)
grafana/grafana: docs/sources/visualizations/panels-visualizations/query-transform-data/_index.md

To save a query you've created:

> 1. From the query editor, open the **Saved queries** dropdown then click the **Save query** option:

Reviewer: ```suggestion
1. From the query editor, open the **Saved queries** drop-down menu and click the **Save query** option:
```

1. To create a query, do one of the following:
   - Write or construct a query in the query language of your data source.
   - Open the **Saved queries** dropdown and click **Replace query** to reuse a saved query.

   {{< admonition type="note" >}}
   [Saved queries](#saved-queries) is currently in [public preview](https://grafana.com/docs/release-life-cycle/). Grafana Labs offers limited support, and breaking changes might occur prior to the feature being made generally available.

   This feature is only available on Grafana Enterprise and Grafana Cloud.
   {{< /admonition >}}

> 1. (Optional) To [save the query](#save-a-query) for reuse, click the **Save query** option in the **Saved queries** dropdown.

Reviewer: ```suggestion
1. (Optional) To [save the query](#save-a-query) for reuse, click the **Save query** option in the **Saved queries** drop-down menu.
```
